pytensor/link/pytorch/dispatch/basic.py

```diff
@@ -0,0 +1,105 @@
+import warnings
+from functools import singledispatch
+
+import torch
+
+from pytensor.compile.ops import DeepCopyOp, ViewOp
+from pytensor.graph.fg import FunctionGraph
+from pytensor.ifelse import IfElse
+from pytensor.link.utils import fgraph_to_python
+from pytensor.raise_op import CheckAndRaise
+
+
+@singledispatch
+def pytorch_typify(data, dtype=None, **kwargs):
+    r"""Convert instances of PyTensor `Type`\s to PyTorch types."""
+    if dtype is None:
+        return data
+    else:
+        return torch.tensor(data, dtype=dtype)
+
+
+@pytorch_typify.register(torch.Tensor)
+def pytorch_typify_tensor(data, dtype=None, **kwargs):
+    # if len(data.shape) == 0:
+    #     return data.item()
+    return torch.tensor(data, dtype=dtype)
```

Comment:
This shouldn't be needed, the default typify will already do the same

Reply:
OK

Reply:
Tried this approach, it did not work on `test_pytorch_FunctionGraph_once`, the numpy array did not get converted to torch tensor